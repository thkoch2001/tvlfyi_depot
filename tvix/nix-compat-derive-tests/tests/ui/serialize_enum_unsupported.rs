use nix_compat_derive::NixSerialize;

#[derive(NixSerialize)]
pub enum Test {
    Foo { version: u8 },
}

fn main() {}
