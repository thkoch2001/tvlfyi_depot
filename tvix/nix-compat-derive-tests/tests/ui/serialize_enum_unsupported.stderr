error: derive(NixSerialize) does not support enums (yet?).
 --> tests/ui/serialize_enum_unsupported.rs:3:10
  |
3 | #[derive(NixSerialize)]
  |          ^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `NixSerialize` (in Nightly builds, run with -Z macro-backtrace for more info)
